### PR TITLE
docs(config): add remote debugging port example

### PR DESCRIPTION
**Pull request by metalix2**
**Summary**

Making it more clear for how you can connect lighthouse to an existing browser.

This Setting isn't obvious and this should help make things clearer for the next person. Wanting to run lighthouse to a remote browser.

**Related Issues/PRs**

https://github.com/GoogleChrome/lighthouse-ci/issues/559

### CODE REVIEW

**Comment by metalix2** [comment]
I just re-read over @patrickhulce comment in https://github.com/GoogleChrome/lighthouse-ci/issues/559 about making the suggestion more about using cypress. I can update to be more specific to Cypress. I've just set it as a more generic remote connection.

**Comment by patrickhulce** [comment]
Thanks so much for following up on this @metalix2! I'll let @connorjclark decide whether it's worth adding to the lighthouse core docs, but in my comment I was referring to the [common LHCI examples](https://github.com/GoogleChrome/lighthouse-ci/blob/main/docs/configuration.md#common-examples) :) sorry for the confusion!

I have a feeling it is a more typical use case for LHCI than a majority of casual Lighthouse users and it is somewhat covered in the [Lighthouse core CLI help](https://github.com/GoogleChrome/lighthouse#cli-options).

File: docs/configuration.md [hunk 1]
```diff
@@ -59,14 +59,15 @@ The `extends` property controls if your configuration should inherit from the de
...
-The settings property controls various aspects of running Lighthouse such as CPU/network throttling and which audits should run.
+The settings property controls various aspects of running Lighthouse such as CPU/network throttling and which audits should run. You can also specify to run lighthouse against a remote browser by passing tbe chrome debugging port.
 
 #### Example
 ```js
 {
   settings: {
     onlyCategories: ['performance'],
     onlyAudits: ['works-offline'],
+    port: 9222,
```
Comment (connorjclark) [inline docs/configuration.md:70]: This should be a separate example. Can you pull out the sentence you added and make a new `js` block immediately following this one?

**Review by connorjclark — CHANGES_REQUESTED** [review]

File: docs/configuration.md (hunk 1 above)
Comment (metalix2) [reply docs/configuration.md:80]: Yeah happy to do so.

**Comment by metalix2** [comment]
> Thanks so much for following up on this @metalix2! I'll let @connorjclark decide whether it's worth adding to the lighthouse core docs, but in my comment I was referring to the [common LHCI examples](https://github.com/GoogleChrome/lighthouse-ci/blob/main/docs/configuration.md#common-examples) :) sorry for the confusion!
> 
> I have a feeling it is a more typical use case for LHCI than a majority of casual Lighthouse users and it is somewhat covered in the [Lighthouse core CLI help](https://github.com/GoogleChrome/lighthouse#cli-options).

Yeah If you'd prefer in LHCI that's fine it was just that the lhci docs referenced these setting options so figured it would be best here. Happy to move to LHCI if you'd prefer that.

**Review by connorjclark — APPROVED** [review]

**Comment by connorjclark** [comment]
@metalix2 can you sign the CLA? thanks!

**Comment by connorjclark** [comment]
@metalix2 bump.

**Comment by connorjclark** [comment]
Can reopen when CLA has been signed.

**Pull request closed without merging**